src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -176,7 +192,7 @@ def session_less_get(
         | ``json``     | A JSON serializable Python object to send in the body of the request. |
         | ``headers``  | Dictionary of HTTP Headers to send with the request. |
         | ``cookies``  | Dict or CookieJar object to send with the request. |
-        | ``files``    | Dictionary of file-like-objects (or ``{'name': file-tuple}``) for multipart encoding upload. ``file-tuple`` can be a 2-tuple ``('filename', fileobj)``, 3-tuple ``('filename', fileobj, 'content_type')`` or a 4-tuple ``('filename', fileobj, 'content_type', custom_headers)``, where ``'content-type'`` is a string defining the content type of the given file and ``custom_headers`` a dict-like object containing additional headers to add for the file. |
+        | ``files``    | Dictionary of file-like-objects (or ``{'name': file-tuple}``) for multipart encoding upload. ``file-tuple`` can be a 2-tuple ``('filename', fileobj)``, 3-tuple ``('filename', fileobj, 'content_type')`` or a 4-tuple ``('filename', fileobj, 'content_type', custom_headers)``, where ``'content-type'`` is a string defining the content type of the given file and ``custom_headers`` a dict-like object containing additional headers to add for the file. List or tuple of ``('key': file-tuple)`` allows uploading multiple files with the same key, resulting in a list of files on the receiving end. |
```

Comment:
the strange thing is that even the official documentation don't mention files as list or tuple..
https://requests.readthedocs.io/en/latest/api/#requests.request

Reply:
Yes, I know that they don't mention it. But the support is there and I don't know how long it has been there.